pkg/config/config.go

```diff
@@ -30,6 +31,17 @@ func (c *Config) Fetch(path string) ([]byte, error) {
 	return f, nil
 }
 
+// Env Fetch env vars from the OCP3 cluster
+func (c *Config) Env(envVar string) (string, error) {
```

Comment:
This is fine for now, but somehow this is bothering me.
This is not directly related to this patch but we've been talking about removing config to env.go.
Maybe we just need placing Env somewhere else.